src/transformers/pipelines/__init__.py

```diff
@@ -623,11 +624,9 @@ def pipeline(
                     alphabet_filename = BeamSearchDecoderCTC._ALPHABET_SERIALIZED_FILENAME
                     allow_regex = [language_model_glob, alphabet_filename]
 
-                    decoder = BeamSearchDecoderCTC.load_from_hf_hub(
-                        pretrained_model_name_or_path, allow_regex=allow_regex
-                    )
+                    decoder = BeamSearchDecoderCTC.load_from_hf_hub(model_name, allow_regex=allow_regex)
                     kwargs["decoder"] = decoder
-                except Exception as e:
+                except ImportError as e:
```

Comment:
A general exception error is too broad here - it didn't catch that `os` wasn't even important

Reply:
No I had it broader because of `kenlm` which might fail too, and IIRC it's not an import error.